internal/cobraprompt/prompt.go

```diff
@@ -146,18 +151,27 @@ func (co CobraPrompt) Run(ctx context.Context, root *cobra.Command, cnfg *hazelc
 			root, _ = rootcmd.New(cnfg)
 			prepareRootCmdForPrompt(co, root)
 			root.SetArgs(promptArgs)
+			root.SetFlagErrorFunc(func(_ *cobra.Command, err error) error {
+				return hzcerror.FlagError(err)
```

Comment:
Do we have to cast `err` to `FlagError`?

Reply:
I believe this is not casting, we are just wrapping the actual error with a custom type, to notify main error handler that this is a flags related error.